packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -301,6 +303,38 @@ export abstract class AbstractMessageManager<
     this.setMessageStatus(messageId, status);
   }
 
+  /**
+   * Updates the message rawData without changing the status.
+   *
+   * @param messageId - The id of the Message to sign.
+   * @param rawData - The data to update rawData in the message.
+   */
+  updateMessageDataInline(messageId: string, rawData: string) {
```

Comment:
This can be abstracted to a new `setResult` which would just set `rawSig`, that should keep it decoupled from any single domain but avoid setting the property directly as it originally did.

Reply:
Done 👍